### PR TITLE
Moamin's edits to Development strategy

### PR DESCRIPTION
**Pull request by Moamin-AbuEwaida**
I'm pushing some edits to the DS file as adding 2 form fields in the HTML file for the toDo task items and Completed items
also I'm proposing the event listener for the checkbox to be "change" instead of "click"

### CODE REVIEW

File: planning/development-strategy.md [hunk 1]
```diff
@@ -20,6 +20,9 @@
 - [ ] *label* tag named add item
 - [ ] *input* tag type="text" for the user to enter an item
 - [ ] *add* button
+- [ ] *form* to have the ToDo list
```
Comment (rayanejsilva) [inline planning/development-strategy.md:23]: Hey, we already mentioned  an input field in **_for: tasks to be done_** (tag li with input type="checkbox")

File: planning/development-strategy.md [hunk 2]
```diff
@@ -20,6 +20,9 @@
...
+- [ ] *form* to have the Completed list
```
Comment (rayanejsilva) [inline planning/development-strategy.md:24]: We mentioned an input field in **_for: completed tasks_** (tag li with input type="checkbox", checked)

Comment (Moamin-AbuEwaida) [reply planning/development-strategy.md:24]: My idea to have both forms existed at the interface of the program, whenever the user adds an item, the form of "toDo" will be increased @rayanejsilva 
Screenshot from 2021-08-17 11-32-03

Comment (rayanejsilva) [reply planning/development-strategy.md:24]: I didn’t understand why we need a tag form for this. We can use a function to append the new items to our ul tag.

**Pull request closed without merging**